Makefile

```diff
@@ -189,10 +189,7 @@ format:
 fmt: format
 
 lint:
-	# tests/utils.go is too long already. Don't make it worse.
-	# this is a placeholder for a proper linter.
-	git diff --numstat $$(git merge-base HEAD origin/main) -- tests/utils.go | \
-		awk '{if ($$1-$$2 > 0) {print $$3 " grew longer"; exit 1} }'
+	if [ $$(wc -l < tests/utils.go) -gt 3565 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
```

Comment:
Can't we use the number of lines of `tests/utils.go` in main instead of hard coded  3565 ?
This way we won't need to update this number every time we reduce the number of lines.

Reply:
We cannot use `git`, that was the previous version of this.
I do not think we need to update it all the time, just occasionally. This job is also not required for merging, so all these are to be use for attention/informative.